lemur/plugins/lemur_aws/elb.py

```diff
@@ -421,11 +422,67 @@ def attach_certificate_v2(listener_arn, port, certificates, **kwargs):
     :param certificates:
     """
     try:
-        return kwargs["client"].modify_listener(
+        needs_cert_update_for_sni = has_listener_cert_for_sni(listener_arn, kwargs["client"])
+
+        modified_listener = kwargs["client"].modify_listener(
             ListenerArn=listener_arn, Port=port, Certificates=certificates
         )
+
+        if needs_cert_update_for_sni:
+            click.echo("[+] Adding cert as listener cert for SNI")
```

Comment:
should we be using current_app.logger instead? it seems that's mostly what we use today for plugins and click.echo is mainly for cli commands